test/framework/modules.py

```diff
@@ -514,7 +514,7 @@ def test_modules_tool_stateless(self):
 
         if isinstance(self.modtool, Lmod):
             # GCC/4.6.3 is nowhere to be found (in $MODULEPATH)
-            load_err_msg = r"The following module\(s\) are unknown"
+            load_err_msg = r"The[\s\n]*following[\s\n]*module\(s\)[\s\n]*are[\s\n]*unknown"
```

Comment:
well, it does

remember, I'm also testing with old Lmod versions, so this is just something we need to deal with (it's also not a big issue imho)

we're not testing Lmod here, we're checking that some module is not available because EB didn't put it there